index.js

```diff
@@ -217,14 +217,12 @@ const create = (win, options) => {
 		if (hasText && props.misspelledWord && props.dictionarySuggestions.length > 0) {
 			dictionarySuggestions = props.dictionarySuggestions.map(word);
 		} else {
-			dictionarySuggestions.push(
-				{
-					id: 'dictionarySuggestions',
-					label: 'No Guesses Found',
-					visible: hasText && props.misspelledWord,
-					enabled: false
-				}
-			);
+			dictionarySuggestions.push({
+				id: 'dictionarySuggestions',
+				label: 'No Guesses Found',
+				visible: hasText && props.misspelledWord,
+				enabled: false
+			});
```

Comment:
Don't do unrelated changes.

Reply:
Yeah, sorry. Related to prettierrc - will disable auto-formatting in my editor.

Reply:
> Don't do unrelated changes.

Reverted